### ISSUE TITLE
Update masterlist.yaml to utilise prelude.yaml concept

### ISSUE THREAD

**Issue by pStyl3** [loot/skyrim#614]
Under https://github.com/loot/loot/issues/1492

In order for the masterlists to make use of the newly created [prelude.yaml](https://github.com/loot/prelude), a few things must be adjusted.
- [ ] Add the new `prelude` node - with it's own `common` sub-node - before the existing masterlist `common` node
```yaml
prelude:
  common:

common:
bash_tags:
globals:
groups:
plugins:
```
- [ ] Add the new short definitions of the Message Anchors under `prelude\common` like [this](https://github.com/loot/loot/issues/1492#issuecomment-922371449)
- [ ] Remove the existing Message Anchors and Sub-Anchors from `masterlist.yaml`'s existing `common` section, as they have been moved to `prelude.yaml`. Only the File Anchors will remain here.
- [ ] In `masterlist.yaml`'s `globals` section, replace the existing localised messages, that have been introduced as new Message Anchors in `prelude.yaml`'s `common` section, by their respective new Sub-Anchors

I might be missing something, but for now I think that these are all the tasks we need to perform on the masterlists.

**Comment by pStyl3** [comment]
The way I currently see it is, that we can't do every single of the above tasks all at once, at least not until a new LOOT version has been released that includes the updated libloot, or not?

We probably can start with adding the `prelude/common` node(s) to the masterlist, including the new short definitions of the Message Anchors.

But once we remove the Message Anchors from `masterlist.yaml`'s `common` node (as they have been moved to `prelude.yaml`), and also try to replace the `masterlist.yaml`'s `globals`'s localised messages with Sub-Anchors (that reference newly defined Message Anchors in `prelude.yaml`'s `common` section) - we try to do something, which the current LOOT v0.16.1 cannot handle.

Is my reasoning wrong here?

**Comment by Ortham** [comment]
> The way I currently see it is, that we can't do every single of the above tasks all at once, at least not until a new LOOT version has been released that includes the updated libloot, or not?

I think you could do almost everything now, if you moved the localised messages in `masterlist.yaml` into the `prelude:` section (so like you have above, but not only the English content strings), because then everything is still self-contained in the masterlist but the localised messages will be replaced by what's in the prelude file when/if that is loaded. Once v0.17 is released, you can then remove the localised strings from the masterlist's prelude section.

That leads to duplicate localisations in the prelude file and the masterlist, but that's always going to be the case for masterlists earlier than v0.17 anyway.

**Comment by pStyl3** [comment]
The Part 1 PR has been merged. If no issues get reported until tomorrow evening, I will do the same for the other masterlists, as well.

**Comment by sibir-ine** [comment]
When moving the global messages to the `prelude` node in the form of new anchors, you should leave behind aliases of those anchors so they're still displayed in the meantime. For instance, there should be a '*latestLOOTThread' in the `globals` node of each masterlist after migration.

**Comment by pStyl3** [comment]
Right, you are correct. If I use my definitions from [here](https://github.com/loot/loot/issues/1492#issuecomment-926833794), so e.g.
```yaml
# Latest LOOT Thread
  - &lootThread
    <<: *latestLOOTThread
    subs: [ 'https://loot.github.io/latest-thread/' ]
```
in `masterlist.yaml`'s `globals` node, the message will be correctly displayed. I will fix that in the masterlists I already updated in a bit.

I've also updated the opening post of this issue to reflect that, the last point for `Part 1` was previously listed for `Part 2`.

Thanks

**Comment by pStyl3** [comment]
`Part 1` of the migration is finished for all masterlists.
